.github/workflows/ci.yml

```diff
@@ -4,12 +4,11 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
```

Comment:
@erunion Assuming we no longer have anything relying on node 12 anymore right? Removing this, b/c one of our packages I think no longer supports 12 and [was failing](https://github.com/readmeio/http-status-codes/actions/runs/4896879669/jobs/8744260963?pr=72)

Reply:
Yea this is good